### ISSUE TITLE
File field fails silently on frontend

### ISSUE THREAD

**Issue by wilrevehl** [WebDevStudios/CMB2#649]
Hey Justin, had a [thread on your blog](https://webdevstudios.com/2015/03/30/use-cmb2-to-create-a-new-post-submission-form/#comment-119575) but figured I mention it here so it's on the radar.

The file field no longer brings up the Media Manager when used on the front-end after WP 4.5.x. I found a [stackexchange conv](http://wordpress.stackexchange.com/questions/224139/add-media-has-stopped-working-in-the-front-end-since-4-5-update )o saying some jQuery deprecations may be the culprit. I tested the bandaid mentioned there and it resolves this. So this may more likely be a WordPress issue. I've yet to figure out cmb._handleMedia, it's doing a ton (nice work). If I get a chance anytime soon to chase the rabbit I'll certainly follow up.

**Comment by tw2113** [comment]
Not managing to recreate the issue with the featured image upload spot from the snippet library/original post.

If someone can manage to recreate, let us know and we'll investigate at that time. Otherwise, closing for the time being.

**Comment by wilrevehl** [comment]
Right on - maybe WordPress resolved it on their end. I had seen the problem
across several similar implementations. Thanks - sounds like good news to
me!

**Comment by jtsternberg** [comment]
the field used for media uploads in the tutorial is a standard text field, modified to be a input of `type="file"`. It is NOT a CMB2 `'file'` field as getting that properly implemented on the front-end is an entirely different beast. A standard `type="file"` field requires no JS/jQuery.

**Comment by wilrevehl** [comment]
Thanks Justin - I did have your CMB2 file field working on the front-end
for logged in visitors with a certain role/permissions. It was fine until a
WP update. I was able to circumvent the issue by downgrading the version of
jQuery to 1.11.3 like mentioned in the stackexchange thread referenced. I
haven't revisited it on the newest of new anything. My guess is something
to do with a bind/targeting approach of an element has changed in jQuery.
